README.md

```diff
@@ -1,20 +1,19 @@
 # command-server README
 
-Creates an http server that exposes a REST api to execute commands. 
+Adds support for running arbitrary commands via file-based RPC.  Designed for
+use with voice-control systems such as [Talon](https://talonvoice.com/).
 
 ## Features
 
-On startup, spawns an http server listening for commands to execute. Each
-running VSCode editor workspace runs its own server, each of which pick a
-different random free port on which to listen.
-The port of the server for the active editor is written to a file named
-`vscode-port` in the operating system's default directory for temporary files.
-
-Accepts requests of the form:
-
-```http
-POST /execute-command HTTP/1.1
+On startup, creates a directory in the default tmp directory, called
+`vscode-command-server-${userName}`, where `${userName}` is the username.  Then
+waits for the `command-server.runCommand` command to be issued, which will
+trigger the command server to read the `request.json` file in the communication
+directory, execute the command requested there, and write the response to
+`response.json`.  
```

Comment:
Do you depend on requests being a single line w/ newline? Waiting until there's a newline in the file is cleaner than repeatedly re-parsing until you hit valid JSON. If so, you should document it here.

Reply:
No that doesn't matter on this side because we just wait for the keystroke to indicate request is ready

Reply:
This readme documents the response side as well. I think it's worth being more explicit about the jsonl format.